src/ParaTest/Logging/JUnit/TestCase.php

```diff
@@ -106,6 +106,27 @@ protected function addDefect($collName, $type, $text)
     }
 
     /**
+     * Add systemOut result on test (if has fail or have error)
```

Comment:
(if it has ~~fail~~ failed)

that should be enough because by phpunit definition an error is also a failure